style.css

```diff
@@ -414,3 +429,293 @@ form {
   align-items: center;
   padding-bottom: 32px;
 }
+
+@media only screen and (min-width: 768px) {
+  /* For Desktop */
+  body {
+    margin: 0;
+    padding: 0;
+    background-image: none;
+    overflow: hidden;
+    overflow-y: scroll;
+  }
+
+  .menu {
+    display: none;
```

Comment:
- [ ] Kindly make sure your page is responsive on the screen size of `768px`. The element should be centered as they are for another screen size. Kindly fix that.
Your design :point_down: 
![Screenshot from 2022-11-03 08-34-15](https://user-images.githubusercontent.com/97100091/199660482-b3a579e4-7c7f-480d-baf5-e253bde0e6c5.png)

![Screenshot from 2022-11-03 08-34-35](https://user-images.githubusercontent.com/97100091/199660496-cdd5675b-9aa5-4196-b0ab-7b0648053cf3.png)